streampark-common/src/main/scala/org/apache/streampark/common/conf/ConfigConst.scala

```diff
@@ -21,21 +21,18 @@ import java.time.LocalDateTime
 object ConfigConst {
 
   /** common const */
-
   val DEFAULT_DATAMASK_STRING = "********"
 
   val PARAM_PREFIX = "--"
 
   /** pyflink */
-
   val PYTHON_SUFFIX = ".py"
 
   val PYTHON_DRIVER_CLASS_NAME = "org.apache.flink.client.python.PythonDriver"
 
   val PYTHON_EXECUTABLE = "venv.zip/venv/bin/python3"
 
   /** about parameter... */
-
   val KEY_APP_HOME = "app.home"
 
```

Comment:
Would you mind checking whether the `SIGN_COLON` `SIGN_SEMICOLON` `SIGN_COMMA` are referenced ?
![image](https://github.com/apache/incubator-streampark/assets/64569824/69496e27-5625-4af3-a6df-e9e21230243f)
If not, It would be better to remove it.

Reply:
ok,I will have a try